apps/sms/style/sms.css

```diff
@@ -232,6 +231,15 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   pointer-events: none;
 }
 
+[data-type="list"] .summary {
```

Comment:
Also I see that long LTR name in RTL or long RTL name in LTR is too close to the contact image [1]. Could you please fix this as well, while we're here?

[1] https://people.mozilla.org/~azasypkin/bug-1138350/issue.png